src/Migrations/Version20240410102808.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240410102808 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Resubmit gift aid claims - see BG2-2595';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $campaignIdForBG2_2595 = "6178";
+
+        $this->addSql(<<<SQL
+            UPDATE Donation
+            SET
+                tbgGiftAidRequestQueuedAt = NULL,
+                tbgGiftAidRequestFailedAt = NULL,
+                tbgGiftAidRequestConfirmedCompleteAt = NULL,
+                tbgGiftAidRequestCorrelationId = NULL,
+                tbgGiftAidResponseDetail = NULL,
+                salesforcePushStatus = 'pending-update'
+            WHERE
+                campaign_id = $campaignIdForBG2_2595 AND
+                giftAid = 1 AND
+                tbgShouldProcessGiftAid = 1 AND
+                tbgGiftAidRequestCorrelationId IS NOT NULL AND
+                donationStatus = 'Paid'
+            LIMIT 73;
```

Comment:
noticed the 73 here was copied from a previous migration, isn't actually right in this case.

Reply:
Running ```
SELECT count(*) from Donation WHERE campaign_id = 6178 AND giftAid = 1 AND tbgShouldProcessGiftAid = 1 AND tbgGiftAidRequestCorrelationId IS NOT NULL AND donationStatus = 'Paid'; 
```

in prod returns 57